lib/dictionary

```diff
@@ -12,6 +12,7 @@ AIME
 Airbnb
 Airbnb's
 Airtable
+alice
```

Comment:
I think the username bit should probably be in an inline code block in that
case (ex. `alice1337`)

On Thu, Jan 21, 2016 at 2:44 PM, Jonathan Leung <notifications@github.com>
wrote:

> In lib/dictionary
> <https://github.com/hackclub/markcop/pull/65#discussion_r50477067>:
>
> > @@ -12,6 +12,7 @@ AIME
> >  Airbnb
> >  Airbnb's
> >  Airtable
> > +alice
>
> See
> https://github.com/hackclub/hackclub/pull/652/files#diff-6438b250a154a96449ce4d3382b1d8b1R50
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hackclub/markcop/pull/65/files#r50477067>.
>



-- 
Zach Latta
https://zachlatta.com <http://zachlatta.com>